sbncode/BeamSpillInfoRetriever/BNBRetriever/BNBRetriever_module.cc

```diff
@@ -0,0 +1,320 @@
+////////////////////////////////////////////////////////////////////////
+// Class:       BNBRetriever
+// Plugin Type: producer 
+// File:        BNBRetriever_module.cc
+//
+// Created by hand Wed April 9 2021 by J. Zennamo (FNAL)
+// Based heavily on code by Z. Pavlovic written for MicroBooNE 
+// Based heavily on code by NOvA collaboration (Thanks NOvA!):
+//       https://cdcvs.fnal.gov/redmine/projects/novaart/repository/entry/trunk/IFDBSpillInfo/BNBInfo_module.cc
```

Comment:
This will be at a github url Soon (TM). Couple of weeks?

Reply:
I will be sure to update it as soon as it is migrated